people/ben-cerveny/index.md

```diff
@@ -0,0 +1,13 @@
+---
+role: director
+---
+
+![ben cerveny](nebBio.jpg)
+
+# Ben Cerveny
```

Comment:
This needs to be on its own line above the photo, otherwise, Jekyll cannot extract your name as the title for the page.